clar2wasm/src/words/tokens.rs

```diff
@@ -352,13 +360,12 @@ impl ComplexWord for TransferNonFungibleToken {
             .i32_const(id_length as i32);
 
         // Push the identifier onto the stack
+        let identifier_ty = generator.nft_types.get(token).cloned().ok_or_else(|| {
+            GeneratorError::TypeError("Usage of nft-burn? on an unknown nft token".to_owned())
```

Comment:
Change `nft-burn?` -> `nft-transfer?`.

Reply:
Tsss, you saw through my copy/pasting scheme